contrib/tlp/patches/01-fix-tlp-stat.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/func.d/tlp-func-stat b/func.d/tlp-func-stat
+index 0eefb0f..a19bffd 100644
+--- a/func.d/tlp-func-stat
++++ b/func.d/tlp-func-stat
+@@ -14,7 +14,7 @@ readonly SESTATUS=sestatus
+ readonly SMARTCTL=smartctl
+ 
+ readonly RE_AC_QUIRK='^UNDEFINED$'
+-readonly RE_ATA_ERROR='ata[0-9]+: SError: {.*CommWake }'
++readonly RE_ATA_ERROR='ata[0-9]+: SError: \{.*CommWake \}'
```

Comment:
what's the purpose of this change?

Reply:
`RE_ATA_ERROR` is supposed to be an extended regexp.

According to [the POSIX spec](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_05_02), unescaped `{` should be followed with a valid `DUP_COUNT`, otherwise results are undefined.

Reading the code, I've deduced that the intention of the braces in this regexp was to match the literal characters themselves. So, I fixed the regexp accordingly.